modules/src/ics04_channel/msgs.rs

```diff
@@ -39,10 +39,47 @@ pub enum ChannelMsg {
     ChannelCloseConfirm(MsgChannelCloseConfirm),
 }
 
+/*
+    `PacketMsg` represents packet messages sent between an ALPHA chain
+    and a BETA chain.
+
+    The roles of ALPHA and BETA are FIXED and never change during
+    the relaying session. DIFFERENT relaying sessions are executed
+    for the case where the ALPHA and BETA roles are FLIPPED
+
+    Depending on the variant, a packet can flow in *either* direction,
+    and the exact flow is documented below.
+
+    All variants are parameterized by the SAME ALPHA and BETA
+    chains, i.e. the chain roles DO NOT FLIP.
+
+    In the same relaying session, the packet's ALPHA chain is
+    ALWAYS the SOURCE chain, and the packet's BETA
+    chain is ALWAYS the DESTINATION chain.
+*/
 #[derive(Clone, Debug, PartialEq)]
 pub enum PacketMsg {
+    /*
+        A `RecvPacket` represents the ALPHA chain RECEIVING
+        an INCOMING packet from the BETA chain.
+     */
     RecvPacket(MsgRecvPacket),
+
+    /*
+        An `AckPacket` represents the ALPHA chain SENDING
+        an acknowlegement as OUTGOING packet to the BETA chain.
```

Comment:
```suggestion
        An `AckPacket` represents the BETA chain SENDING
        an acknowlegement as OUTGOING packet to the ALPHA chain.
```

Reply:
I would avoid using the terms "OUTGOING" and "INCOMING". 
As I mentioned before, all ics04 packet messages include a `Packet` that is identical for all the messages for a given channel and packet sequence. So here, if we consider A the source chain (where `packet_src_channel` and `packet_src_port` are) and B is the destination chain (where `packet_dst_channel` and `packet_dst_port`) are.

```suggestion
        An `AckPacket` relays information to the source chain (A) about the destination chain (B) writing
        an acknowlegement for the specified packet included in the message.
```